src/app/permission/app-search/app-search.component.html

```diff
@@ -1,7 +1,8 @@
-<ocx-portal-page helpArticleId="PAGE_PERMISSION_SEARCH">
+<ocx-portal-page permissionArticleId="PAGE_PERMISSION_SEARCH">
```

Comment:
mmmh, I'm quite sure that change is not correct

Reply:
It should be this: <ocx-portal-page permission="PAGE_PERMISSION_SEARCH"> ?